solver/demos/curtiss_hirschfelder.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 PONIO TEAM. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+#include <cmath>
+#include <iostream>
+#include <numbers>
+#include <numeric>
+
+#include <solver/butcher_methods.hpp>
+#include <solver/observer.hpp>
+#include <solver/problem.hpp>
+#include <solver/solver.hpp>
+
+/*
+solve Curtiss and Hirschfelder problem:
+
+$$
+    \begin{aligned}
+        \dot{y} =  k(\cos(t) - y) \\
+        y(0) = y_0
+    \end{aligned}
+$$
+
+$y_0 = 2$
+*/
+
+int
+main( int, char** )
```

Comment:
Why do you add `int, char**` since it's not used?

Reply:
Because I was hoping to use them...